crates/agent/src/agent.rs

```diff
@@ -0,0 +1,16 @@
+/// Generic Aries cloudagent python structure
```

Comment:
Do we still need to have the "python" bit here?
```suggestion
/// Generic Aries cloudagent structure
```